### DISCUSSION TITLE
GitHubからのSSOでアイコンが更新されない

### DISCUSSION THREAD

**Discussion by kattakaha** [increments/qiita-discussions#880] [要望・アイデア・不具合]
GitHubからSSOでQiitaに登録した場合にGitHubのアイコンを変更してもQiitaに変更が反映されていないような気がする。
cacheが効いている可能性があったのでブラウザcacheを削除したり半年近く時間をおいても反映されていないので、何かしら不具合があるのではないかと思い報告させていただきます。

![image](https://github.com/increments/qiita-discussions/assets/58053010/6f42db7e-93e2-4c56-84da-b2b86f16f3ea)

GitHub該当アカウントリンク：https://github.com/kkml4220
Qiita該当アカウントリンク：https://qiita.com/kkml_4220

**Comment by kattakaha** [comment]
> GitHubやX(Twitter)でユーザー名を変更した場合は、いったんQiitaから [ログアウト](https://qiita.com/logout) して再度OAuth認証を行なってください

とありますが、これをやっても変更が反映されない

**Reply by tanaka-asami415** [reply]
@kkml4220 
ご報告ありがとうございます。ご不便をおかけして申し訳ありません🙇

確認したところ、弊社で利用しているCDNサービスのキャッシュが原因のようです。
本日、キャッシュ削除を試したのですが現時点で反映されておりません。
追加で調査し、新しいことが分かりましたら改めてご報告いたします。

**Reply by kattakaha** [reply]
@tanaka-asami415 
半年近くTTLが設定されているのが不思議ですが原因究明のきっかけになれて光栄です。
別に困ったことはないので美味しいコーヒーでも飲みながらゆっくり修正していただければと思います。

**Comment by wataru86** [comment]
@kkml4220

お待たせしました！キャッシュを削除し、更新された画像が表示されることを確認しました。
環境によっては以前の画像が表示される場合があるため、お手元の環境でも更新された画像が表示されるか確認していただきたいです🙏 

![image (18)](https://github.com/user-attachments/assets/16bcd242-4bb6-44ee-bfbb-5af3ebc7fa1a)

**Reply by kattakaha** [reply]
@wataru86 @tanaka-asami415 
始業したとたんにいいニュースでした！また、変更されていることも観測しました。おそらく正常に動作していると思われます。

改めて対応ありがとうございました。よい一日を！

![image](https://github.com/user-attachments/assets/65c6f5af-ae78-441c-a792-75642c678d3e)

**Reply by tanaka-asami415** [reply]
@kkml4220 
こちらこそ、ご確認いただきありがとうございます！
問題が解決されたとのこと、何よりです✨
